doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -381,18 +381,27 @@ Example python code to replace a file.  This may be run by changing these parame
 Builtin Users
 ~~~~~~~~~~~~~
 
-This endpoint deals with users of the built-in authentication provider. For more background on various authentication providers, see :doc:`/user/account` and :doc:`/installation/config`.
+Builtin users are known as "Username/Email and Password" users in the :doc:`/user/account` of the User Guide. Dataverse stores a password (encrypted, of course) for these users, which differs from "remote" users such as Shibboleth or OAuth users where the password is stored elsewhere. See also "Auth Modes: Local vs. Remote vs. Both" in the :doc:`/installation/config` section of the Installation Guide. It's a valid configuration of Dataverse to not use builtin users at all.
```

Comment:
Do we encrypt passwords? If so, we really should be using a better system (hash+salt)